glossary.rst

```diff
@@ -0,0 +1,46 @@
+.. _glossary:
+
+========
+Glossary
+========
+
+This page contains definitions for Flux-specific terms and concepts mentioned
+throughout the various guides and docs on our sites.
+
+------------------------------------------------------------------------------
+
+**Flux Broker**
+  A distributed message broker daemon that provides communications services
```

Comment:
I would say something in layman's terms like:

> You can imagine the broker as the main controller of the Flux instance - the one running the show, and ensuring that X, Y, Z.

(Metaphors are generally great for docs, when you take a concept and turn it into a mini story so it sticks better).

Reply:
I like this suggestion a lot, thanks! It might be helpful for me to get an example of what _X,Y,Z_ might be in this metaphor (maybe the flux-core guys can help me out here).

Reply:
haha, I put letters because I'm not totally sure. I could guess based on the flux operator, but best to ask. If it's helpful here is the template I use:

```yaml
# Flux needs to know the path to the IMP executable
[exec]
imp = "/usr/libexec/flux/flux-imp"

[access]
allow-guest-user = true
allow-root-owner = true

# Point to resource definition generated with flux-R(1).
[resource]
path = "/etc/flux/system/R"

[bootstrap]
curve_cert = "/etc/curve/curve.cert"
default_port = 8050
default_bind = "tcp://eth0:%%p"
default_connect = "tcp://%%h.%s:%%p"
hosts = [
	{ host="%s-%s"},
]
```